pom.xml

```diff
@@ -181,30 +193,29 @@
             <!-- override kafka-clients transitive dependency for 0.10 needed   -->
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka-clients</artifactId>
-            <version>0.10.2.2</version>
+            <version>${kafka-clients.version}</version>
         </dependency>
         <dependency>
             <!-- update from version in transient dependency of ch-kafka -->
             <groupId>org.xerial.snappy</groupId>
             <artifactId>snappy-java</artifactId>
-            <version>1.1.10.5</version>
+            <version>${snappy-java.version}</version>
         </dependency>
         <dependency>
             <!-- update from version in transient dependency of ch-kafka -->
             <groupId>org.sonarsource.scanner.maven</groupId>
             <artifactId>sonar-maven-plugin</artifactId>
-            <version>3.9.1.2184</version>
-        </dependency>
-        <dependency>
-            <!-- update from version in transient dependency of sonar-maven-plugin -->
-            <groupId>org.sonarsource.scanner.api</groupId>
-            <artifactId>sonar-scanner-api</artifactId>
-            <version>2.16.3.1081</version>
-        </dependency>
-        <dependency>
-            <groupId>uk.gov.companieshouse</groupId>
-            <artifactId>kafka-models</artifactId>
-            <version>${kafka-models.version}</version>
+            <version>${sonar-maven-plugin.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.codehaus.plexus</groupId>
+                    <artifactId>plexus-sec-dispatcher</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.sonarsource.scanner.api</groupId>
+                    <artifactId>sonar-scanner-api</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
```

Comment:
Can we add a version for this to make sure its latest?

Reply:
can not add version to exclusion